docs/localization/LOCALIZATION.md

```diff
@@ -1,52 +1,56 @@
 
-- Custom LocalizationConfiguration can be injected from the application
+- Support for resource keys with parameters.
 
-- MissingTranslationsService - tries to fetch key without context if contextualized key not defined in the bundle
-
-- Intelligent fallback support. For any un-supported language request, localization service fallback to fallback locale map language OR first nearest supported language. More information in the `intelligent fallback` section below.
+- Custom `LocalizationConfiguration` can be injected from the application.
 
 - Ability to toggle translations on and off to help identify the key corresponding to a translation.
 
 # How to use
 
-We provide in [library](https://github.com/AmadeusITGroup/otter/blob/main/packages/@o3r/localization/src/tools/localization.module.ts) an Angular module called **LocalizationModule** which comes with translations loader.
+We provide an Angular module in __@o3r/localization__ called [LocalizationModule](https://github.com/AmadeusITGroup/otter/blob/main/packages/@o3r/localization/src/tools/localization.module.ts) which comes with a translations loader.
 
-- **In your AppModule** you need to **import** the **LocalizationModule** and **TranslateModule**. The LocalizationModule could be imported calling `forRoot` with a custom configuration **factory** to specify the language of the application. This configuration is of type **LocalizationConfiguration** and describes your endpoint URL, supported locales, list of RTL languages, the language of your application and your fallback language.
+## Configuration of Localization Module
 
-```typescript
+In your `AppModule`, you need to **import** the `LocalizationModule` and `TranslateModule` (from __@ngx-translate/core__). The `LocalizationModule` could be imported calling its `forRoot` method
+with a custom configuration **factory** to specify the language of the application. This configuration is of type `LocalizationConfiguration` and describes:
+- The endpoint URL - where to fetch the translation bundles from
+- Supported locales - list of available languages
+- RTL languages - list of RTL language codes
+- The language of the application - display language
+- The fallback language - fallback in case the translation in the selected language does not exist
 
-// index.ts
+All the possible configuration options can be found in the `LocalizationConfiguration` [interface](https://github.com/AmadeusITGroup/otter/blob/main/packages/%40o3r/localization/src/core/localization.configuration.ts).
 
-import { LocalizationModule, translateLoaderProvider } from  "@o3r/localization";
+You can also import the `LocalizationModule` without calling its `forRoot` method. In this case, you will be provided with some default `LocalizationConfiguration`.
```

Comment:
I think `forRoot` should always be called in the `AppModule` but you are not forced to provide a configuration, if any then a default configuration will be applied